stetho-okhttp3/src/main/java/com/facebook/stetho/okhttp3/StethoInterceptor.java

```diff
@@ -223,7 +223,7 @@ public boolean connectionReused() {
 
     @Override
     public int connectionId() {
-      return mConnection.hashCode();
+      return mConnection == null ? 0 : mConnection.hashCode();
```

Comment:
Add `@Nullable` to the `mConnection` field and the `connection` constructor parameter too, please :)